UnitTest1.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MICP_Assignments;
+using System.Collections.Generic;
+
+namespace UnitTestsForAssignment3
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        private int[] inputArrayForTest = new int[6];
+        private int[][] testArray1 = new int[2][];
+
+        [TestInitialize]
+        public void arrayInitializationForTesting()
+        {

```

Comment:
Populating the array this way is definitely valid. If you're looking to save space while maintaining readability, you can also do the following: inputArrayForTest = {-1, 0, 1, 2, -1, -4};